IncBackups/IncBackupsControl.py

```diff
@@ -921,24 +909,21 @@ def DeleteSnapShot(self, inc_job):
             self.website = job_snapshots[0].job.website.domain
             self.externalApp = job_snapshots[0].job.website.externalApp
             self.passwordFile = '/home/%s/%s' % (self.website, self.website)
-
             for job_snapshot in job_snapshots:
 
-                ## Functions above use the self.jobid varilable to extract information about this snapshot, so this below variable needs to be set
-
+                ## Functions above use the self.jobid variable to extract information about this snapshot, so this below variable needs to be set
                 self.jobid = job_snapshot
+                self.jobids = self.jobids + job_snapshot.snapshotid + ' '
```

Comment:
this is where I concatenate all of the jobids into one string, which is the passed only 1 time to the delete function. Considerable time savings